### DISCUSSION TITLE
Custom categories

### DISCUSSION THREAD

**Discussion by Exzellius** [TomBursch/kitchenowl#605] [Q&A] [answered]
Hey there,

simple small question I couldnt find an answer for. Is there the possibility of creating custom categories?

Best regards
Ex

**Comment by irishrain** [comment, marked as answer]
Yes, go to Settings, Household Settings, Categories. You can add, edit and sort categories there.

**Reply by Exzellius** [reply]
Beautiful, thanks. I am still getting used to the UI.

**Comment by ParysLouis** [comment]
And would it be possible to add an icon next to our new category?

Thanks in advance :)

**Reply by TomBursch** [reply]
The icons are just emojis (if you edit the name of a default one you can see that). So for a new category you can just use another emoji
